scripts/scripts/testing/automations/batch/lebanon.py

```diff
@@ -0,0 +1,21 @@
+import pandas as pd
+from bs4 import BeautifulSoup
+import requests
+from datetime import date
+
+url = 'https://corona.ministryinfo.gov.lb'
+req = requests.get(url)
+soup = BeautifulSoup(req.text, 'html.parser')
+
+count = int(data.find_all('h1',attrs={'class':'s-counter-3 s-counter'}).text)
+date = str(date.today())

```

Comment:
Avoid using `date` (it shadows library module) as the variable name, proposal: use `date_str`.

Reply:
We already have a script for Lebanon so let's stick with that one